### PR TITLE
[RDM]More loadout stuff

### PR DESCRIPTION
**Pull request by improvedname**
more loadout stuff...

:cl: Improvedname
add: more loadout stuff
/:cl:

### CODE REVIEW

File: modular_citadel/code/modules/client/loadout/uniform.dm [hunk 1]
```diff
@@ -89,3 +148,9 @@
 	category = slot_w_uniform
 	path = /obj/item/clothing/under/rank/head_of_security/skirt
 	restricted_roles = list("Head of Security")
+
+/datum/gear/turtleneck
+	name = "Tactitool Turtleneck"
+	category = slot_w_uniform
+	path = /obj/item/clothing/under/syndicate/tacticool
```
Comment (deathride58) [inline modular_citadel/code/modules/client/loadout/uniform.dm]: That's 50% fire resistance and 40% acid resistance. Make a variant that doesn't have any protective qualities

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]

File: modular_citadel/code/modules/clothing/under.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/obj/item/clothing/under/syndicate/cosmetic
```
Comment (cebutris) [inline modular_citadel/code/modules/clothing/under.dm:1]: You might want to remove the `NO_SENSORS` flag from this variant

Comment (improvedname) [reply modular_citadel/code/modules/clothing/under.dm:1]: Feature

Comment (cebutris) [reply modular_citadel/code/modules/clothing/under.dm:1]: ok

**Comment by Poojawa** [comment]
Eventually we'll need to separate by job categories for some of this stuff.

**Comment by improvedname** [comment]
gotcha

**Comment by improvedname** [comment]
Ill do that in the next loadout related pr

**Pull request merged**